scl/default-network-drivers/plugin.conf

```diff
@@ -27,9 +27,9 @@ block source default-network-drivers(
 	rfc5424-tls-port(6514)
 	rfc5424-tcp-port(601)
 	log-msg-size(65536)
-	flags()
-	tls()
-	hook-commands()
+	flags(_EMPTY_)
```

Comment:
The idea is great, but i don't like the token _EMPTY_. Can't we simply use the quote characters?

eg. flags("") means that it's optional and the default is an empty string.

Reply:
The only downside I see is when an option does not expect string, it looks strange to pass "" to it, for example in case of hook-commands or tls. Other than that, I do not see any technical reason why shouldn't this work.

I will try this out and report back if I run into any trouble with it.

Reply:
hmm.. you have a point here. The "" may not be the best token after all.

Just le me try to think about it:
  * port() -- the argument is required
  * port(anything) -- the argument has a default and is not required
  * port("") -- the argument defaults to an empty string and is not required

And the last one is what we are still debating, right? Clearly if port is assumed to be numeric, this might look strange. My argument on the other side is that if something indeed can be an empty string, then it's actually a string and not anything else.

I am ok with this format, are you?

Reply:
Yes, we need some token to express empty string. If we can reuse `""`, it is less awkward than having an `_EMPTY_` token for the same purpose. The thing is the suggested version does not work seamlessly, I ran into trouble with an scl, which contained prefix(`prefix`), which was expanded to prefix(), hence configuration error. I could workaround this instance by writing `prefix("`prefix"`)` instead. I'll try to patch all similar cases and see if any other problem occurs.